plugin.py

```diff
@@ -128,40 +133,47 @@ def is_valid_for_view(self, view: sublime.View) -> bool:
     def _handle_log_message_notification(self, payload: CopilotPayloadLogMessage) -> None:
         pass
 
+    @notification_handler(NTFY_PANEL_SOLUTION)
+    def _handle_panel_solution_notification(self, payload: CopilotPayloadPanelSolution) -> None:
+        view = ViewPanelCompletionManager.find_view_by_panel_id(payload["panelId"])
+        if not view:
+            return
+
+        preprocess_panel_completions(view, [payload])
+
+        completion_manager = ViewPanelCompletionManager(view)
+        completion_manager.append_completion(payload)
+        completion_manager.update()
+
+    @notification_handler(NTFY_PANEL_SOLUTION_DONE)
+    def _handle_panel_solution_done_notification(self, payload) -> None:
+        view = ViewPanelCompletionManager.find_view_by_panel_id(payload["panelId"])
+        if not view:
+            return
+
+        ViewPanelCompletionManager(view).is_waiting = False
```

Comment:
With `reset`, it could be more readable `ViewPanelCompletionManager(view).reset()`

Reply:
Done in [a168798](https://github.com/TheSecEng/LSP-copilot/pull/18/commits/a16879844cf092d263701cb321475b3e143e8044)